docs/src/guides/smartweave/warp/deploying-contracts.md

```diff
@@ -0,0 +1,106 @@
+# Warp: Deploying SmartWeave Contracts
+
+SmartWeave Contracts are made up of two transactions, a Source Transaction and a Initial State Transaction, the source transaction contains the source code the contract will use to determine the current state. The initial state transaction provides a contract identifer to reference as well as the initial seed data the contract should use as the starting point to evaluate the current state. The current state is calculated by accessing actions that are transactions written to the network that contain input parameters to execute using the evaluated and instanciated source code. Warp Contracts can be created using many different languages and can be evaluated using the Warp SDK. This guide will show the many different ways you can deploy a Warp Contract.
+
+::: tip
+If you would like to learn more about authoring Warp SmartWeaveContracts, checkout the Warp Academy! [https://academy.warp.cc/](https://academy.warp.cc/)
+:::
+
+## The Four ways to deploy a Warp SmartWeave Contract
+
+There are 4 ways you can deploy a SmartWeaveContract via the Warp SDK, these options handle different use cases that a developer may encounter.
+
+* Need to deploy the contract with the source at the same time
+* Need to deploy a contract where the source is already on the permaweb
+* Need to deploy a contract through the sequencer and point it to some data using a path manifest
+* Need to deploy a contract via bundlr and register that contract on the sequencer
+
+::: tip
+For more information about Warp deployments check out the github Readme for the project. [https://github.com/warp-contracts/warp#deployment](https://github.com/warp-contracts/warp#deployment). 
+:::
+
+::: warning
+This project is in rapid development, so the documentation here could be out of data quickly, if you discover it is out of date, please let us know on the [discord channel](https://discord.gg/haCAX3shxF).
```

Comment:
recommend specifying the channel name (wether this is pointing to warp or cookbook channel)